.github/workflows/unittests.yaml

```diff
@@ -12,7 +12,7 @@ jobs:
       - name: Use Node.js
         uses: actions/setup-node@v3
         with:
-          node-version: '20.5.x'
+          node-version: '20.10.x'
```

Comment:
Same nit as above, looks like we used generic "20.10" above in e2e_cron.yml#L24 and e2e_pr.yml#L35, but now adding `.x` patch semver operator.

Not sure if we want to pin everything to an exact major.minor.patch so half our CI doesn't update with patches and the other half doesn't.